src/SonarScanner.MSBuild.Common/CommandLine/CommandLineFlagPrefix.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
```

Comment:
Missing copyright

Reply:
... and some redundant `using` statements.

Reply:
Agreed, fixed.